src/app/core/layers/imagelayer.js

```diff
@@ -171,8 +171,8 @@ proto.getWmsUrl = function({ type = 'map' } = {}) {
   return this.config.wmsUrl;
 };
 
-proto.getWFSLayerName = function(){
-  return this.getQueryLayerName().replace(/[/\s]/g, '_')
+proto.getWFSLayerName = function() {
+  return this.getQueryLayerName().replace(/\s+/g, '_').replace( ':', '-' );
```

Comment:
```suggestion
  return this.getQueryLayerName().replace(/\s+/g, '_').replaceAll( ':', '-' );
```

Reply:
@Raruto i think that /g mean reaplce all (globally)

Reply:
@volterra79 for the ":" character you didn't use a regex, just a simple: `replace( ':', '-' );`